src/resources/Pipelines/StatementGroups/StatementGroups.ts

```diff
@@ -35,7 +38,7 @@ export default class StatementGroups extends Resource {
         );
     }
 
-    update(pipelineId: string, groupId: string, groupModel: StatementGroupModel) {
+    update(pipelineId: string, groupId: string, groupModel: UpdateStatementGroupModel) {
```

Comment:
StatementGroupModel required fields that were not considered on the update. I updated this.